db/db.js

```diff
@@ -401,6 +447,34 @@ async function getLocationIDFromParticleCoreID(coreID) {
   }
 }
 
+async function getRadarTypeFromLocationID(locationid) {
+  try {
+    const results = await pool.query('SELECT (radar_type) FROM locations WHERE locationid = $1', [locationid])
+    if (results === undefined) {
+      return null
+    }
+
+    return results.rows[0].radar_type
+  } catch (e) {
+    helpers.log(`Error running the getLocationData query: ${e}`)
+  }
+}
+
+// Retrieves the locationid corresponding to a particle device coreID
+async function getLocationFromParticleCoreID(coreID) {
+  try {
+    const results = await pool.query('SELECT * FROM locations WHERE door_particlecoreid = $1 OR radar_particlecoreid = $1', [coreID])
+    if (results === undefined) {
+      helpers.log('Error: No location with associated coreID exists')
+      return null
+    }
+
+    return results.rows[0]
```

Comment:
Might be nice to use `createLocationFromRow` to return a `Location` object instead. Up to you.